### ISSUE TITLE
Table of public vs. privately held debt

### ISSUE THREAD

**Issue by liaochris** [liaochris/SPEOC-pt-1#13]
@sanjeev0813 let me know once you've accepted the invitation. Can you summarize what we discussed over slack here? We can use github to discuss so that our notes are easier to track down. 

The data you should be using is [here](https://github.com/liaochris/SPEOC-pt-1/blob/main/data_clean/final_data_CD.csv). Take a look at lines 3360-3363 for examples of what I mean by "publicly held debt". Publicly held debt will either by people who are government officials or "government sound names" like Commonwealth of Pennsylvania. 

To figure out whose a govenrment official, here are a few tips
1) their occupation is often considered "treasurer"
2) They often hold a lot of debt even though they're not personally very wealthy (Benjamin Harwood from MD is my favourite example)
3) you'll probably have seen their name come up in the context of being a public official in power of the purse. 

I know we discussed making a table, but you should feel free to use whatever way of visualizing the data you think is most informative.

**Comment by sanjeev0813** [comment]
yea sure - we wanted to figure out how much of the debt was held by the government/states and how much was held by private citizens in each state. the end product would be a two column table where rows would be states and columns would be privately/publicly held debt. values would be the quantity of debt.

**Comment by liaochris** [comment]
If you have qs, tag me in the thread and i'll respond here.

**Comment by sanjeev0813** [comment]
will do. ill start working in a jupyter notebook for now and will send that to you or add it into the github once im done

**Comment by liaochris** [comment]
Sounds good. please summarize what you've done in this thread when you have major updates.

**Comment by liaochris** [comment]
If Isabella has bandwidth, please loop her into what's going on as well.

**Comment by liaochris** [comment]
@Izzyy2003 - issue we're using for discussion.

**Comment by liaochris** [comment]
@sanjeev0813  can you create a folder called speculation and move your [notebook](https://github.com/liaochris/SPEOC-pt-1/blob/main/Public%20Debt%20-%20Hamilton%201790.ipynb) there? Also, in your commit messages, if you include #13, it'll automatically link the commit to the issue which helps me keep track of it.

**Comment by Izzyy2003** [comment]
> yea sure - we wanted to figure out how much of the debt was held by the government/states and how much was held by private citizens in each state. the end product would be a two column table where rows would be states and columns would be privately/publicly held debt. values would be the quantity of debt.

ok thanks

**Comment by sanjeev0813** [comment]
I just finished most of my work. What I did: I got a table of all the public debt in each state by finding the names of treasurers in the [dataset](https://github.com/liaochris/SPEOC-pt-1/blob/main/data_clean/final_data_CD.csv) using Political Graveyard. I then found the debts of each treasurer along with the State of __ and Commonwealth of __ Debt values within the dataset. I've added a [ReadMe](https://github.com/liaochris/SPEOC-pt-1/blob/main/debt%20analysis/README.md), a [PDF with all the Treasurer Names](https://github.com/liaochris/SPEOC-pt-1/blob/main/debt%20analysis/Names%20of%20Treasurers%20in%20Data.pdf), the [Jupyter Notebook I worked on](https://github.com/liaochris/SPEOC-pt-1/blob/main/debt%20analysis/Public%20Debt%20-%20Hamilton%201790.ipynb), and a [CSV file with the table](https://github.com/liaochris/SPEOC-pt-1/blob/main/debt%20analysis/debt_summary.csv) that go more in-depth into what I did.

**Comment by liaochris** [comment]
Updates made by Chris
1. Removed Jedediah Huntington from CT and David Rittenhouse from PA - treasurers in 1789 so outside of our range of interest
2. Removed treasurers where we saw an entry for "state of/commonwealht of" - for example, Peter Colt, treasurer of connecticut, was removed because we saw a state of connecticut entry
    - affected individuals are: Peter Colt, CT, John Gibbons, GA, John Meals, GA, John Taylor Gilman, NH, Nathaniel Gilman, NH, Oliver Peabody, NH, William Gardner, NH, Christian Febiger, PA, Thomas Taylor, SC
3. Gerard Bancker served as treasurer for 20 years -> [see here](https://en.wikipedia.org/wiki/New_York_State_Treasurer)
